tests/integration-tests-topologies/src/main/java/org/apache/pulsar/tests/topologies/PulsarClusterTestBase.java

```diff
@@ -56,19 +57,15 @@
     protected static PulsarCluster pulsarCluster;
 
     @BeforeClass
-    public static void setupCluster() throws Exception {
-        StringBuilder sb = new StringBuilder();
-        for (int i = 0; i < 8; i++) {
-            sb.append((char) (ThreadLocalRandom.current().nextInt(26) + 'a'));
-        }
+    public void setupCluster() throws Exception {
         PulsarClusterSpec spec = PulsarClusterSpec.builder()
-            .clusterName(sb.toString())
+            .clusterName(this.getClass().getSimpleName())
```

Comment:
why do you change the logic here?

Reply:
It will identify the containers for individual tests.

Reply:
I would prefer keeping the randomized string, you can prefix with class name.

Reply:
randomization does not give us any benefits us.

Reply:
it has - if you tied to class name, that means you can only have one test running with that given class. in a CI environment, this will tend to break. if you want to improve the identification part, use the class name + the existing randomized string.

Reply:
will add some random characters